persist/fs/index_read_write_go-pre-1-10_test.go

```diff
@@ -0,0 +1,27 @@
+// +build !go1.10
```

Comment:
can we drop this and only use go1.10 and up?

Reply:
Sounds good.